package.json

```diff
@@ -24,10 +24,13 @@
     "minikube-deploy": "./bin/minikube-deploy",
     "minikube-services": "./bin/minikube-services",
     "minikube-services-all": "./bin/minikube-services-all",
+    "prepare-awscli": "./bin/prepare-awscli",
+    "prepare-gcloud": "./bin/prepare-gcloud",
     "prepare-kubectl": "./bin/prepare-kubectl",
     "run-blocking-job": "./bin/run-blocking-job",
     "timeout.sh": "./bin/timeout.sh",
-    "verify-deployment": "./bin/verify-deployment"
+    "verify-deployment": "./bin/verify-deployment",
+    "cloud-login": "./bin/cloud-login"
```

Comment:
Please add the `cloud-login` higher up to keep this alphabetical, unless there is a reason this should be kept last.